Sources/Public/LottieConfiguration.swift

```diff
@@ -32,6 +36,22 @@ public struct LottieConfiguration: Hashable {
   /// The decoding implementation to use when parsing an animation JSON file
   public var decodingStrategy: DecodingStrategy
 
+  /// Use the display P3 color space for rendering instead of the default RGB
+  ///  - Defaults to `false`
+  public var useDisplayP3: Bool
```

Comment:
Rather than having a boolean for toggling Display P3, could we have a `animationColorSpace: CGColorSpace` property? That API would be more flexible since it lets consumers use any color space. We could include instructions / sample code in the doc comment on how to set the color space to Display P3.

Reply:
Done. I can follow up with a PR on the doc repo but basically this can be used like so:
```
if let displayP3 = CGColorSpace(name: CGColorSpace.displayP3) {
    LottieConfiguration.shared.colorSpace = displayP3
}
```

Reply:
Thanks!